app/src/main/java/org/dosomething/letsdothis/ui/CampaignDetailsActivity.java

```diff
@@ -134,7 +134,7 @@ public void onUserClicked(String id)
         //FIXME
         if(BuildConfig.DEBUG)
         {
-            startActivity(PublicProfileActivity.getLaunchIntent(this));
+            startActivity(PublicProfileActivity.getLaunchIntent(this ,id));
```

Comment:
crashes because the api is return bad data. works sometimes if you click the correct user..